src/csexp_rpc/csexp_rpc.ml

```diff
@@ -228,8 +228,20 @@ module Server = struct
     let loop () =
       let* accept = accept () in
       match accept with
-      | Error _
+      | Error `Stopped ->
+        Log.info [ Pp.text "RPC stopped accepting." ];
+        Fiber.return None
+      | Error (`Exn exn) ->
+        Log.info
```

Comment:
Maybe `Log.error` here? (if that's even a thing)

Reply:
It's not a thing unfortunately.

Reply:
Why not crash the process completely here? Having a dune server that continues running but doesn't accept connections seems ... not great?

Reply:
If there's a pending build, I can see a good case for completing the build and *then* shutting down.

Reply:
On the other hand, if it's an half an hour build the user might be wondering why the key bindings they are typing in their editor have no effect.

In any, accept errors might be rare enough that this is low-priority.

Reply:
It also seems bad to immediately terminate while there are still active sessions. But if you prefer this behavior, I can do that.